HISTORY.txt

```diff
@@ -364,4 +364,5 @@
                      Add CLI support for performing final sanity check for ExDB loading and holdings in etl.load_ex.DbLoadingWorkflow task;
                      Update CI/CD testing to use python 3.10
  23-Dec-2024  V1.726 Skip integers that exceed max int32 in DataTransformFactory
-  7-Jan-2025  V1.727 Handle "None" values in vrpt data
\ No newline at end of file
+  7-Jan-2025  V1.727 Handle "None" values in vrpt data
+  22-jan-2024 V1.727 Add Imgs format option (for jpg/svg generation) to splitIdList()
```

Comment:
I think you mean 2025, not 2024 :). Also, could you update the version number and adjust the spacing so it lines up with the line above it?

```suggestion
 22-Jan-2025 V1.728 Add Imgs format option (for jpg/svg generation) to splitIdList()
```